hass/devices.js

```diff
@@ -26,33 +26,40 @@ const FAN_DIMMER = {
 const THERMOSTAT_2GIG = {
   type: 'climate',
   object_id: 'thermostat',
-  values: ['49-1-1', '64-1-0', '66-1-0', '67-1-1', '67-1-2', '68-1-0'],
+  values: [
+    '49-1-Air temperature',
+    '64-1-mode',
+    '66-1-state',
+    '67-1-Heating',
+    '67-1-Cooling',
+    '68-1-mode'
+  ],
   mode_map: {
-    off: 'Off',
-    heat: 'Heat',
-    cool: 'Cool'
+    off: 0,
+    heat: 1,
+    cool: 2
   },
   fan_mode_map: {
-    auto: 'Auto Low',
-    on: 'On Low'
+    auto: 0,
+    on: 1
   },
   setpoint_topic: {
-    Heat: '67-1-1',
-    Cool: '67-1-2'
+    Heat: '67-1-Heating',
+    Cool: '67-1-Cooling'
   },
-  default_setpoint: '67-1-1',
+  default_setpoint: '67-1-Heating',
   discovery_payload: {
     min_temp: 50,
     max_temp: 85,
     modes: ['off', 'heat', 'cool'],
     fan_modes: ['auto', 'on'],
-    action_topic: '66-1-0',
+    action_topic: '66-1-state',
     action_template: '{{ value_json.value | lower }}',
-    current_temperature_topic: '49-1-1',
+    current_temperature_topic: '49-1-Air temperature',
```

Comment:
I believe this value becomes either `49-0-Air temperature` or `49-0-Air_temperature` 
And this for all commandClasses 49. 
from my mqtt, path is: `zwave2mqtt/Office/RR/49/0/Air_temperature`


Json:
```
{"id":"76-49-0-Air temperature","nodeId":76,"commandClass":49,"commandClassName":"Multilevel Sensor","endpoint":0,"property":"Air temperature","propertyName":"Air temperature","type":"number","readable":true,"writeable":false,"label":"Air temperature","genre":"user","unit":"°C","list":false,"value":18.58,"lastUpdate":1606159264282}
```

Reply:
> from my mqtt, path is: zwave2mqtt/Office/RR/49/0/Air_temperature

Tht's correct because mqtt removes spaces from topics and replace them with `_`. It's not the same to use `Air_temperature` and `Air temperature` in valueId, the first is wrong